java/src/test/android/README.md

```diff
@@ -20,7 +20,8 @@ This android application is mainly aimed for testing:
 ### Requirements
 
 - JDK version 8 or later is required.
```

Comment:
change to JDK 11. Java 8 doesn't work with the later version of the Android Gradle plugin.